openmetadata-spec/src/main/resources/json/schema/entity/data/worksheet.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/data/worksheet.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Worksheet",
+  "$comment": "@om-entity-type",
+  "description": "This schema defines the Worksheet entity. A Worksheet is a tab or sheet within a spreadsheet file (Google Sheets, Excel) that contains structured data.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.entity.data.Worksheet",
+  "javaInterfaces": [
+    "org.openmetadata.schema.EntityInterface"
+  ],
+  "properties": {
+    "id": {
+      "description": "Unique identifier of this worksheet instance.",
+      "$ref": "../../type/basic.json#/definitions/uuid"
+    },
+    "name": {
+      "description": "Name of the worksheet.",
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "fullyQualifiedName": {
+      "description": "Fully qualified name of the worksheet.",
+      "$ref": "../../type/basic.json#/definitions/fullyQualifiedEntityName"
+    },
+    "displayName": {
+      "description": "Display Name that identifies this worksheet.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of the worksheet.",
+      "$ref": "../../type/basic.json#/definitions/markdown"
+    },
+    "spreadsheet": {
+      "description": "Reference to the parent File entity (with fileType=Spreadsheet)",
+      "$ref": "../../type/entityReference.json"
+    },
+    "service": {
+      "description": "Link to the drive service",
+      "$ref": "../../type/entityReference.json"
+    },
+    "serviceType": {
+      "description": "Type of drive service",
+      "$ref": "../../entity/services/driveService.json#/definitions/driveServiceType"
+    },
+    "worksheetId": {
+      "description": "Native worksheet/tab ID",
+      "type": "string"
+    },
+    "index": {
+      "description": "Position/index of the worksheet in the spreadsheet",
+      "type": "integer"
+    },
+    "rowCount": {
+      "description": "Number of rows in the worksheet",
+      "type": "integer"
+    },
+    "columnCount": {
+      "description": "Number of columns in the worksheet",
+      "type": "integer"
+    },
+    "columns": {
+      "description": "Column definitions if structured data",
```

Comment:
just one last thought, should we allow also to bring in information about a worksheet charts as well? entityRef to `chart` entity, since most spreadsheets might have both data and visualizations